app/controllers/api/v1/comments_controller.rb

```diff
@@ -1,27 +1,33 @@
-class Api::V1::CommentsController < Api::V1::ApiController 
-  
-    before_action :authenticate_token!
-    
-    def index
-      post_id = params[:post_id]
-      @comments = Post.find(post_id).comments
+class Api::V1::CommentsController < Api::V1::ApiController
+  skip_before_action :authenticate_token!
+
+  def index
+    post_id = params[:post_id]
+    @comments = Post.find(post_id).comments
+
+    if @comments.empty?
+      render json: { status: 'ERROR', message: 'No Comments Found' }, status: :not_found
+    else
       render json: @comments, status: :ok
     end
-  
-    def create
-      author_id = @current_user.id,
-      text = params[:text],
-      post_id = params[:post_id]
-      if author_id && text && post_id
-        comment = Comment.new(author_id:, text:, post_id:,)
-        if comment.save
-          render json: { status: 'SUCCESS', message: 'Comment Has Been Created', data: comment }, status: :ok
-        else
-          render json: { status: 'ERROR', message: 'Comment Has NOT Been Created', data: comment .errors}, status: :unprocessable_entity
-        end
-      else
-        render json: { status: 'ERROR', message: 'Invalid Prameters' }, status: :not_found
-      end
+  end
+
+  def create
+    @comment = Comment.create(
+      text: comment_params[:text],
+      author_id: params[:author_id],
+      post_id: params[:post_id]
+    )
+    if @comment.save
+      render json: @comment, status: :created
+    else
+      render json: { status: 'ERROR', message: 'Comment Not Created' }, status: :unprocessable_entity
     end
   end
-  
\ No newline at end of file
+
+  private
+
+  def comment_params
+    params.permit(:text)
+  end
```

Comment:
While trying to use the ```Post - /users/:user_id/posts/:id/comments```, I could not post a comment. Kindly check that you are permitting the appropriate data to complete the post request.

Reply:
Hello @chrystalme. Thank you for your review. I would like to show you that the post comment is working well. Below is a screenshot of the result. I tested it after seeing this review.
![adddsadsadadsadsadasdadsa](https://user-images.githubusercontent.com/83514256/206855178-f504c13e-45c3-43e7-9f01-e5300d3ad2e5.PNG)